src/components/cylc/gscan/GScan.vue

```diff
@@ -102,13 +102,13 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
       </div>
       <!-- data -->
       <div
-        v-if="!isLoading && sortedWorkflows && sortedWorkflows.length > 0"
```

Comment:
No more `sortedWorkflows` computed property. When GScan deltas are received, they are updated in Vuex via callbacks registered in the `WorkflowService`. When the Vuex state changes, it triggers a `computed` property that creates the hierarchical elements, adding elements in-order into the tree. Finally watchers will be notified and update the `filteredWorkflows` (i.e. apply filters, if any).